jena-text/src/main/java/org/apache/jena/query/text/TextQueryPF.java

```diff
@@ -224,12 +234,16 @@ private QueryIterator resultsToQueryIterator(Binding binding, Node s, Node score
     }
 
     private QueryIterator variableSubject(Binding binding, Node s, Node score, Node literal, StrMatch match, ExecutionContext execCxt) {
+        if (log.isTraceEnabled())
+            log.trace("variableSubject: {}", match) ;
```

Comment:
You don't need to guard with `isTraceEnabled()` if you are using this form of `Logger.trace`.

Reply:
I'm happy to amend by removing the unnecessary `log.isTraceEnabled()`. I was intending to follow the coding practices elsewhere in TextQueryPF and TextIndexLucene, should I amend the unneeded occurrences of `log.isDebugEnabled()` as well?